### DISCUSSION TITLE
Chromebook Updated.

### DISCUSSION THREAD

**Discussion by 300blkout** [3kh0/ext-remover#56] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My Chromebook was updated by accident what should I do now? Does this mean I can no longer uses this? Will there be a update to bypass the patch? 

I did: Try to use the other method,and check my policy. None of these show any signs of it being blocked. Just seems like Chromebook update patched this.

**Comment by 3kh0** [comment]
What version are you on?

**Reply by 300blkout** [reply]
106.0.5249.1`12

**Reply by 3kh0** [reply]
Try using ALL methods, see if they work, they should not have been patched just yet.

**Reply by 300blkout** [reply]
![image](https://user-images.githubusercontent.com/52010092/196015164-58222ee2-f389-4f49-aa03-b165825f1689.png)

Even on my home pc it is not disabling.

**Reply by 300blkout** [reply]
All admin installed extensions seem to not be disabled regardless of the method.

**Reply by 3kh0** [reply]
Make sure this is chromeos

**Reply by 300blkout** [reply]
I tried it on both chrome os and chrome on my home PC. Neither work on GoGuardian or CIPA filter. No methods are working on admin extensions.

**Comment by phillipbruh** [comment]
Try using the code in bookmark.js if it works mark me as the answer

**Reply by ghost** [reply]
that's what original poster was doing

**Reply by phillipbruh** [reply]
@yeeteeyt 
no he was using ingot

**Reply by 300blkout** [reply]
@phillipbruh No. I said multiple times I tried all methods. It was simply an update [106]. You need to follow [chrome100.dev](https://chrome100.dev/) to fix this. Until a new method is found.

**Comment by 300blkout** [comment, marked as answer]
I tried all methods. It was simply an update [106]. You need to follow [chrome100.dev](https://chrome100.dev/) to fix this.

**Reply by KerbalOfficial** [reply]
but [chrome100.dev](https://chrome100.dev/) is blocked, can you tell me the steps there are in there? would be really helpful.

**Reply by 300blkout** [reply]
Yeah I can do that. 
![image](https://user-images.githubusercontent.com/52010092/197946157-c2c758e8-395f-46ca-b2a3-cd14159e1a0b.png)
You will most likely need a home pc for this and 8gb usb.

**Comment by Fallenmoon8080** [comment]
You could do `CTRL+ALT+T` and open crosh. Type `rollback` and when prompted, type Y and `ENTER` BEWARE, BACK UP EVERYTHING TO GOOGLE DRIVE AND TURN OFF SYNC. In addition, this will clear your search history. Here is an image: 
![image](https://user-images.githubusercontent.com/112455501/197318088-6ad7e298-2a2c-48b0-95d6-5a85f3ba2b1c.png)

**Reply by KerbalOfficial** [reply]
this doesn't work. it says not attempting a rollback, the admins in my chromebook set it to automatically saying **no**

**Reply by Fallenmoon8080** [reply]
unfortunate.

**Reply by 300blkout** [reply]
Unfortunately most administrators are smart enough to block crosh so I would say this is pretty much ineffective for most.
